bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubHandler.java

```diff
@@ -28,7 +28,7 @@
 import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 import org.eclipse.smarthome.core.types.Command;
-
+import org.openhab.binding.neohub.internal.NeoHubBindingConstants.NeoHubReturnResult;
```

Comment:
why is this imported?

Reply:
It's added by the auto save actions because it's used by `fromNeoHubFetchPollingResponse` on L181.